configs/neptune.yaml

```diff
@@ -54,6 +54,8 @@ parameters:
   installments__last_k_trend_periods: '[10, 50, 100, 500]'
   installments__last_k_agg_periods: '[1, 5, 10, 20, 50, 100]'
   installments__last_k_agg_period_fractions: '[(5,20),(5,50),(10,50),(10,100),(20,100)]'
+  pos_cash__last_k_trend_periods: '[6, 12]'
+  pos_cash__last_k_agg_periods: '[6, 12, 30]'
```

Comment:
@kstrzala have you grid searched those values?

Reply:
It depends on what you mean by 'grid searched'. I did not perform any 'classical' grid search, but in the first version I put here many more values and finally left only some, which had any impact on the models' predictions..

Reply:
this version of grid search is completely fine :)